nasa-gibs/worldview
# WV-3148: Update NRT Flood layer descriptions

## Description

Fixes #WV-3148

Updated layer descriptions for Flood (2-day Window) and Flood (3-day window) (the descriptions are the same)

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,MODIS_Combined_Flood_3-Day(disabled=3),MODIS_Combined_Flood_2-Day(disabled=3),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-17-T15%3A04%3A35Z)
2. Read to ensure it's comprehendible and changes have been made [compared to production](https://worldview.earthdata.nasa.gov/?l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,MODIS_Combined_Flood_3-Day(disabled=3),MODIS_Combined_Flood_2-Day(disabled=3),VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-17-T15%3A04%3A35Z)
3. Note: the exclamation notification will be removed for these layers once this PR goes into prod.


@nasa-gibs/worldview
